src/components/Card.jsx

```diff
@@ -8,7 +13,37 @@ export default function Card({ chatName, chatLink, from, to, dateFrom, dateTo, m
         redirectWindow.focus();
     }
 
-    console.log(dateFrom)
+    dayjs.extend(customParseFormat);
+    dayjs.extend(localizedFormat);
+    function formatDate(stringDate) {
+        if (stringDate === null) return null;
+        const date = dayjs(stringDate, 'DD-MM-YYYY');
+        return date.locale('ru').format('D MMMM YYYY');
+    }
+
+    function rewardInfo() {
```

Comment:
If you want to create it as a function - name it as an action
Like getRewardInfo